consensus/pow/sha256worker.go

```diff
@@ -0,0 +1,64 @@
+package consensus
+
+import (
+    "bytes"
+    "crypto/sha256"
+    "math"
+    "math/big"
+)
+
+// data isathe pointer to the target block data,
+// target is the goal, which means we need to mine a result and its hash must be less than the target.
+type SHA256Worker struct {
+    Data   []byte
```

Comment:
Comment is required for all exported type/variable, and better to follow the golint  guideline.

Reply:
nice.